tests/unit/modules/network/ios/test_ios_route_maps.py

```diff
@@ -259,7 +259,7 @@ def test_ios_route_maps_replaced(self):
                                         exact_match=True,
                                         name=["new_replace"],
                                     ),
-                                    ip=dict(address=dict(acls=[10, 100])),
+                                    ip=dict(address=dict(acls=[10, 20, 30])),
```

Comment:
Can you add another replaced test with a slightly different config, maybe one that is similar to the issue tagged!

Reply:
I’ve added extra test cases with a slightly different setup, similar to the tagged issue. The integration looks fine, and the problem seems to be resolved. Let me know if you need any further changes.